Feat/137_create_calendar_skeleton_ui

nkabembo opened the pull request:
Final Calendar Skeleton

enoble450 committed:
Resolve index

enoble450 committed:
Got month to display correctly

enoble450 committed:
commit

enoble450 committed:
commit

enoble450 committed:
commit

enoble450 committed:
commit

enoble450 committed:
commit

nkabembo committed:
Changes for pop up window

enoble450 committed:
information

enoble450 committed:
commit

enoble450 committed:
commit

enoble450 committed:
don't know what happened there

nkabembo committed:
Completed skeleton calendar layout

nkabembo committed:
Merge branch 'feat/137_create_calendar_skeleton_ui' of https://github.com/COSC-470-Okanagan-College/AstroCal into feat/137_create_calendar_skeleton_ui

nkabembo reviewed (commented):
Completed the calendar ui skeleton page

FreddieBanfield commented on AstroCal/Calendar.py:
```diff
@@ -0,0 +1,85 @@
+from kivy.app import App
```
calendar.kv and Calendar.py should both be in the view folder. Please may you move them to the view folder and make necessary import changes if need be.

nkabembo replied:
I have moved files as requested

FreddieBanfield commented on AstroCal/calendar.kv:
```diff
@@ -0,0 +1,189 @@
+<DayMode>:
+	Label: 
```
Having both files named Calendar can be confusing and cause issues when importing, maybe change the .kv file to be called Calendar_layout or something a little more descriptive

nkabembo replied:
.kv file must be name same as app class ,that is the naming convention as the program looks for a kv file named calendar.kv since the class is called CalendarApp.We have however renamed the calendar.py file to be called calendar_ui.py for there to be a distinction

FreddieBanfield commented on AstroCal/Calendar.py:
```diff
@@ -0,0 +1,85 @@
+            return 29
+        else:
+            return 28
+def getRiseSet(year, month, day, celestial, status):
```
getRiseSet is a function in the controller and a lot of this code should be in the core.py. I will have a look at this during my free time tomorrow and I am happy to make the necessary changes if you want me to.

nkabembo replied:
Have deleted getRiseSet from the calendar_ui.py file

FreddieBanfield commented on AstroCal/Calendar.py:
```diff
@@ -0,0 +1,85 @@
+from kivy.app import App
+from kivy.uix.button import Button
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.gridlayout import GridLayout
```
Make sure to merge the latest version of dev into your branch. When I pulled down this branch I noticed it is a few pull merges behind the dev branch.

nkabembo replied:
Have merge latest changes from dev into branch

FreddieBanfield reviewed (changes requested):
Please review all these changes. I am happy to complete these changes if you want me to. Just let me know.

nkabembo committed:
Merge branch 'dev' into feat/137_create_calendar_skeleton_ui

nkabembo committed:
Refactored Code
- Added calendar_ui.py and calendar.kv file to view
- Changed name of calendar.py file to calendar_ui.py
- Remove old tkinker ui files
- Calling calendar-ui in core.py to run view

FreddieBanfield reviewed (approved):
Looks great!
